web/frontend/legacy/lib/ui/content/export_modal.js

```diff
@@ -51,15 +53,17 @@ function downloadFile (includeAnnotations, experimental=false, aws_lambda=false,
       + (includeAnnotations === "true" ? '?annotations=true' : '?annotations=false')
       + (experimental ? '&experimental=true' : '')
       + (aws_lambda ? '&aws_lambda=true' : '')
-      + (docx_footnotes ? '&aws_lambda=false&docx_footnotes=true' : ''));
+      + (docx_footnotes ? '&aws_lambda=false&docx_footnotes=true' : '')
+      + (docx_sections ? '&docx_sections=true' : ''));
   }
 }
 
-function showExportModal (e, experimental=false, aws_lambda=false, docx_footnotes=false) {
+function showExportModal (e, experimental=false, aws_lambda=false, docx_footnotes=false,
+                           docx_sections=false) {
   new ExportModal('export-modal', e.target, {
     'click #export-modal': (e) => { if (e.target.id === 'export-modal') this.destroy()},
     'click .export': (e) => {
-      downloadFile(e.target.value, experimental, aws_lambda, docx_footnotes);
+      downloadFile(e.target.value, experimental, aws_lambda, docx_footnotes, docx_footnotes);
```

Comment:
Should the send `docx_footnotes` be a `docx_sections`?

Reply:
Whoops— think that's in a stash

Reply:
set